### ISSUE TITLE
Build task hangs

### ISSUE THREAD

**Issue by jackpimbert** [betwo/vscode-catkin-tools#25]
Hi,

I recently had the extension working well, however I have encountered an issue where the build task hangs from within vscode. I can run `catkin build` successfully from the terminal, but when I try and run in vscode I get the following:

![cb](https://user-images.githubusercontent.com/3306938/100993293-8385ac00-3555-11eb-97b6-2894d7254144.png)

The task just stays the same, showing no output and I have to terminate it. I can also run the same command in the terminal (e.g. `catkin build --workspace "..."` successfully).

Do you have any idea what could be causing this? Or if there's a way to debug?

I'm running catkin-tools version 1.19.0 and vscode version 1.51.1 on Ubuntu 18.04, with ROS Melodic.

**Comment by jackpimbert** [comment]
Looks like it's an issue with the vscode integrated terminal. I fixed by adding the following lines to `~/.config/Code/User/settings.json`:

```
    "terminal.integrated.inheritEnv": true,
    "terminal.integrated.shell.linux": "/bin/bash",
```

See:
https://stackoverflow.com/questions/54092486/visual-studio-code-terminal-blank-screen

**Comment by betwo** [comment]
Hi @jackpimbert, just out of curiosity: Are both of those options required for the extension to work with your setup?
I guess that means your default shell is not `bash`?
If I could reproduce this on my machines, we would be able to add a notification message to warn other users with similar issues.

**Comment by jackpimbert** [comment]
Hi @betwo,

Good question. I don't believe both are..

I can reproduce toggling the following line in my `~/.config/Code/User/settings.json` file:

```
    "terminal.integrated.inheritEnv": true
```
- With `inheritEnv` set to `true`, I can build via vscode.
- With `inheritEnv` set to `false`, I cannot build via vscode.

The shell option does not make a difference, so that can be removed. I've tried without it, with "zsh" (which is the shell I use) and the behaviour is the same. Note, that I used "bash" in the original comment as an example, but did have "zsh" there in my own setup.

Hope this helps for reproducing.
